frontend/src/pages/admin/Template/sections/Environment.vue

```diff
@@ -186,11 +184,6 @@ export default {
             this.editable.settings.env.splice(index, 1)
         }
     },
-    mounted () {
-        this.editable.settings.env.forEach(field => {
```

Comment:
Are you able to help me understand why this check no longer needs to run on mount?